### ISSUE TITLE
Peer Dependency issues for @nestjs/common@"^10.0.0"

### ISSUE THREAD

**Issue by BayBreezy** [notiz-dev/nestjs-prisma#60]
Hello,

I tried installing with the updated version of Nestjs but it failed.

```bash
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: my-app@0.0.1
npm ERR! Found: @nestjs/common@10.0.1
npm ERR! node_modules/@nestjs/common
npm ERR!   @nestjs/common@"^10.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @nestjs/common@"^7.0.0 || ^8.0.0 || ^9.0.0" from nestjs-prisma@0.20.0
npm ERR! node_modules/nestjs-prisma
npm ERR!   nestjs-prisma@"*" from the root project
```

**Comment by matteovava** [comment]
Same problem and the install with the "nest add" command is not working too
```
▹▹▹▸▹ Package installation in progress... ☕
Failed to execute command: npm install --save nestjs-prisma@latest
✖ Package installation in progress... ☕
Unable to install library nestjs-prisma because package did not install. Please check package name.
```

**Comment by marcjulian** [comment]
Please try out the latest dev release

```bash
npm i nestjs-prisma@0.21.0-dev.2
```

**Comment by BayBreezy** [comment]
Everything works fine when I use yarn instead of npm.. think I will be using yarn from this point forward

**Comment by marcjulian** [comment]
This has been released with [0.21.0](https://github.com/notiz-dev/nestjs-prisma/releases/tag/v0.21.0)

**Comment by BayBreezy** [comment]
Thanks for the hard work :)
